client/deposits/overview/instant-deposit-button.js

```diff
@@ -29,34 +66,68 @@ const InstantDepositModal = ( { amount, fee, net, onClose, onSubmit, inProgress
 					currency.formatCurrency( net / 100 )
 				) }
 			</Button>
+			<Button isDefault onClick={ onClose }>
+				{ __( 'Cancel', 'woocommerce-payments' ) }
+			</Button>
 		</Modal>
-	)
-}
+	);
+};
+
+const InstantDepositSubmitNotice = ( { error } ) => {
+	let message = '';
+	// TODO: supply proper error notices.
+	switch ( error.code ) {
+		case 'balance_insufficient':
+			message = __(
+				'Balance is not sufficient for instant deposit.',
+				'woocommerce-payments'
+			);
+			break;
+
+		default:
+			message = __(
+				'There was an error, please try again.',
+				'woocommerce-payments'
+			);
+	}
+
+	return (
+		<Notice status="error" isDismissible={ false }>
+			<p>
+				<strong>{ message }</strong>
+			</p>
+		</Notice>
+	);
+};
 
 // TODO: Properly style :allthethings:
```

Comment:
I guess you can remove this `TODO` now 😄

Reply:
Done.